src/wheel/bdist_wheel.py

```diff
@@ -167,6 +181,7 @@ def wheel_dist_name(self):
         return '-'.join(components)
 
     def get_tag(self):
+        from . import pep425tags
```

Comment:
Here is where I need some help from the packaging people: how can we replace the use of pep425tags: `get_abbr_impl()`, `get_impl_ver()`, `get_abi_tag()`, `get_supported()`

Reply:
Looking at how pip transitioned is probably best as @pradyunsg pointed out, but a quick guess is:
* `get_abbr_impl()` -> `implementation_name()`
* `get_impl_ver()` -> `implementation_version()`
* `get_abi_tag()` -> No direct equivalent for _just_ that
* `get_supported()` -> `sys_tags()` or proper combination of the other `*_tags()` functions depending on the situation

Reply:
Thanks. It seems packaging might want to support `get_python_tag`, `get_abi_tag` and `get_platform_tag`, as well as `get_platform` (mentioned in a comment above).